twitter_scraper.rb

```diff
@@ -18,6 +18,11 @@ def set_scraping_url(id)
 
   def obtain_analytics_data(agent, export_url, bundle_url)
     agent.post(export_url)
+    for i in 1..10 do
+      res = agent.get(bundle_url)
+      break unless res.body == ""
+      sleep(5)
+    end
```

Comment:
I encounter the case where we failed to get data.

![2018-08-31 15 42 00](https://user-images.githubusercontent.com/35832220/44897166-bbc63080-ad35-11e8-826e-700bc9a547cc.png)

So, I add loop process.

Reply:
[IMO] Should TwitterScraper class do retry of request several times? I think it is better that`obtain_analytics_data` requests data once simply.

```ruby
for i in 1..10 do
  body = twitter_scraper.obtain_analytics_data(agent, export_url, bundle_url)
  break unless body.empty?
  sleep(5)
end
```

Reply:
[IMO] I should know why failure to get data occurs sometimes. But if I can't understand, sending several times is needed.